### PR TITLE
CEDS-1479 Rename ExportsCacheModel to ExportsDeclaration

### PR DESCRIPTION
**Pull request by ed0906**
Huge amount of changes so I'll highlight the key changes

### CODE REVIEW

File: app/models/ExportsDeclaration.scala [hunk 1]
```diff
@@ -24,7 +24,7 @@ import models.declaration.{Locations, Parties, TransportInformationContainerData
 import play.api.libs.json._
 import services.cache.ExportItem
 
-case class ExportsCacheModel(
+case class ExportsDeclaration(
```
Comment (ed0906) [inline app/models/ExportsDeclaration.scala]: Key change

File: test/services/cache/ExportsDeclarationBuilder.scala [hunk 2]
```diff
@@ -26,175 +26,175 @@ import forms.declaration.additionaldeclarationtype.AdditionalDeclarationTypeSupp
 import forms.declaration.destinationCountries.DestinationCountries
 import forms.declaration.officeOfExit.OfficeOfExit
 import forms.{Choice, Ducr}
-import models.ExportsCacheModel
+import models.ExportsDeclaration
 import models.declaration.{DeclarationAdditionalActorsData, DeclarationHoldersData, Locations, TransportInformationContainerData}
 
 //noinspection ScalaStyle
-trait ExportsCacheModelBuilder {
+trait ExportsDeclarationBuilder {
```
Comment (ed0906) [inline test/services/cache/ExportsDeclarationBuilder.scala]: Key Change

File: test/services/cache/ExportsDeclarationBuilder.scala [hunk 3]
```diff
@@ -26,175 +26,175 @@ import forms.declaration.additionaldeclarationtype.AdditionalDeclarationTypeSupp
...
     createdDateTime = LocalDateTime.of(2019, 1, 1, 0, 0, 0),
     updatedDateTime = LocalDateTime.of(2019, 2, 2, 0, 0, 0),
     choice = Choice.AllowedChoiceValues.StandardDec
   )
 
-  private type CacheModifier = ExportsCacheModel => ExportsCacheModel
+  private type ExportsDeclarationModifier = ExportsDeclaration => ExportsDeclaration
 
-  def aCacheModel(modifiers: CacheModifier*): ExportsCacheModel =
+  def aDeclaration(modifiers: ExportsDeclarationModifier*): ExportsDeclaration =
```
Comment (ed0906) [inline test/services/cache/ExportsDeclarationBuilder.scala]: Key change

File: test/services/cache/ExportsItemBuilder.scala [hunk 4]
```diff
@@ -18,58 +18,51 @@ package services.cache
...
 
   private def uuid: String = UUID.randomUUID().toString
 
   private val modelWithDefaults: ExportItem = ExportItem(id = uuid)
 
-  private type CachedItemModifier = ExportItem => ExportItem
+  private type ItemModifier = ExportItem => ExportItem
 
-  def aCachedItem(modifiers: (CachedItemModifier)*): ExportItem =
+  def anItem(modifiers: (ItemModifier)*): ExportItem =
```
Comment (ed0906) [inline test/services/cache/ExportsItemBuilder.scala]: Key change

File: test/services/cache/ExportsDeclarationBuilder.scala (hunk 3 above)
Comment (PatrykRudnicki) [reply test/services/cache/ExportsDeclarationBuilder.scala]: I'm still thinking about value of this `a` inside `aDeclaration`

File: test/services/cache/ExportsItemBuilder.scala (hunk 4 above)
Comment (PatrykRudnicki) [reply test/services/cache/ExportsItemBuilder.scala]: The same here about `an`

**Comment by ed0906** [comment]
Apologies in advance for the inevitable conflicts this will cause

**Pull request merged**